history/taskana-simplehistory-provider/src/test/java/acceptance/AbstractAccTest.java

```diff
@@ -83,14 +87,19 @@ public static WorkbasketHistoryEvent createWorkbasketHistoryEvent(
 
   protected static void resetDb(String schemaName) throws Exception {
     DataSource dataSource = DataSourceGenerator.getDataSource();
+    String schemaNameTmp =
+        schemaName != null && !schemaName.isEmpty()
+            ? schemaName
+            : DataSourceGenerator.getSchemaName();
+    try (Connection connection = dataSource.getConnection()) {
+      String dbProductId =
+          DB.getDatabaseProductId(connection.getMetaData().getDatabaseProductName());
```

Comment:
Here would be another usage for `DB#getDatabaseProductid(Connection)` ;) Maybe it's worth it

Reply:
done